### PR TITLE
Feature/cta popup

### PR DESCRIPTION
**Pull request by GarageInc**
**Summary**

CTA popup for studio
https://favro.com/organization/bdbb4f24afc48b29c74f4fa4/cffeb6e3eaf254588cb68a9a?card=San-7529

**Screenshots**
image
image
image
image

### CODE REVIEW

**Comment by Partysun** [comment]
@GarageInc Text is changed. You can check in Favro

File: src/components/CtaJoinPopup/CtaJoinPopup.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      onOpen={() => setOpen(true)}
+      onClose={() => setOpen(false)}
+      classes={styles}
+    >
+      <Panel padding className={styles.container}>
+        <img src={joinImg} alt='Join' />
+
+        <div className={styles.join}>Join the community!</div>
+        <div className={styles.description}>
+          Santiment is a behavior analytics platform for cryptocurrencies,
```
Comment (Partysun) [inline src/components/CtaJoinPopup/CtaJoinPopup.js]: Change text pls

File: src/assets/join.svg [hunk 2]
```diff
@@ -0,0 +1 @@
...1.445.445.334 1 .779 1.668 1.223.333.223.444.723.222 1.056l-.5 1c-1.334 2.502-1.89 3.558-2.89 3.78-.056.112-.167.112-.278.112zM74.154 28.126c0-3.335 2.557-7.06 6.281-7.06 1.279 0 2.001.779 2.001 2.113 0 .277-.055.555-.055.833 0 .223-.056.445-.056.667 0 .89.778 1.279 1.056 1.279.834 0 1.501-1.279 2.168-2.557.945-1.834 2.112-4.113 4.614-4.113.833 0 1.5.722 1.5 1.556 0 .167 0 .39-.055.611 0 .187-.056.369-.056.556.89-.11 1.668-.945 2.39-1.723M65.871 16.397c-.167 0-.334-.055-.445-.11a17.46 17.46 0 0 0-1.056-.668c-1.278-.778-2.446-1.556-3.724-2.279-.39-.222-.556-.723-.334-1.112.056-.166.223-.278.334-.333.056-.167.167-.278.334-.39 1.612-1.945 2.334-3.112 2.723-3.89.39-.667.667-1.056 1.223-1.168.556-.11.89.223 1.39.612.611.556 1.723 1.5 4.224 2.78a.86.86 0 0 1 .445.61c.055.278 0 .5-.167.723-.056.056-4.169 4.836-4.28 4.947-.111.111-.222.223-.39.278h-.277zM83.548 8.06c-.89 0-1.723-.39-2.335-1.001-.667-.556-1.111-1.445-1.111-2.502C80.102 2.834 81.324 1 83.103 1h.556c1.667 0 3.28 1.779 3.28 3.613.055 2.112-1.724 3.446-3.391 3.446zM87.994 55.53l.5.166c1.557.556 5.17 1.835 6.837 1.835.167 0 .334 0 .556-.056" stroke="#181B2B" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```
Comment (haritonasty) [inline src/assets/join.svg]: Make this with dynamic colors for night mode
(Try to make screenshots for PR in 3 versions for such ui tasks: day & night mode for desktop, mobile) for decreasing future small fixes

File: src/components/CtaJoinPopup/CtaJoinPopup.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import React, { useState, useEffect } from 'react'
+import { Link } from 'react-router-dom'
+import { connect } from 'react-redux'
+import { PATHS } from '../../App'
+import Button from '@santiment-network/ui/Button'
+import Dialog from '@santiment-network/ui/Dialog'
+import joinImg from './../../../src/assets/join.svg'
+import Panel from '@santiment-network/ui/Panel'
+import { checkIsLoggedIn } from '../../pages/UserSelectors'
```
Comment (haritonasty) [inline src/components/CtaJoinPopup/CtaJoinPopup.js]: rearrange imports

File: src/components/CtaJoinPopup/CtaJoinPopup.js (hunk 1 above)
Comment (GarageInc) [reply src/components/CtaJoinPopup/CtaJoinPopup.js]: Yes. Asked Ibis about it but forgot to change :D

File: src/ducks/Studio/index.js [hunk 4]
```diff
@@ -14,6 +14,7 @@ import { useTimeseries } from './timeseries/hooks'
 import { buildAnomalies } from './timeseries/anomalies'
 import { trackMetricState } from './analytics'
 import styles from './index.module.scss'
+import CtaJoinPopup from '../../components/CtaJoinPopup/CtaJoinPopup'
```
Comment (Partysun) [inline src/ducks/Studio/index.js]: @GarageInc it should be before scss.

**Pull request merged**